### PR TITLE
Apply template and auth details to OperationDefinition

### PR DESCRIPTION
**Pull request by daliboz**
Applied the template to the OperationDefinition resource

### CODE REVIEW

File: content/millennium/dstu2/operation-definition.md [hunk 1]
```diff
@@ -7,17 +7,67 @@ title: OperationDefinition | DSTU 2 API
 * TOC
 {:toc}
 
+## Overview
+
+The OperationDefinition resource defines an [operation](http://hl7.org/fhir/dstu2/operations.html). This includes the inputs, outputs, description, and other items necessary to actually execute the operation. 
+
+This resource only exposes the custom operations our server defines. Operations that are defined as part of another implementation guide are referenced directly instead of being duplicated within this server. While the HL7 FHIR specification also supports describing named queries with this resource, we do not currently support any custom named queries.
```
Comment (MaxPhilips) [inline content/millennium/dstu2/operation-definition.md]: Trademarks

File: content/millennium/dstu2/operation-definition.md [hunk 2]
```diff
@@ -7,17 +7,67 @@ title: OperationDefinition | DSTU 2 API
...
+* [URL](http://hl7.org/fhir/dstu2/operationdefinition-definitions.html#OperationDefinition.url){:target="_blank"}
+* [Informal name](http://hl7.org/fhir/dstu2/operationdefinition-definitions.html#OperationDefinition.name){:target="_blank"}
+* [Formal name](http://hl7.org/fhir/dstu2/operationdefinition-definitions.html#OperationDefinition.code){:target="_blank"}
+* [Status](http://hl7.org/fhir/dstu2/operationdefinition-definitions.html#OperationDefinition.status){:target="_blank"}
+* [Kind](http://hl7.org/fhir/dstu2/operationdefinition-definitions.html#OperationDefinition.kind){:target="_blank"}
+* [Name of publisher](http://hl7.org/fhir/dstu2/operationdefinition-definitions.html#OperationDefinition.publisher){:target="_blank"}
+* [Date published](http://hl7.org/fhir/dstu2/operationdefinition-definitions.html#OperationDefinition.date){:target="_blank"}
+* [Description](http://hl7.org/fhir/dstu2/operationdefinition-definitions.html#OperationDefinition.description){:target="_blank"}
+* [Whether or not the operation be invoked without a Resource](http://hl7.org/fhir/dstu2/operationdefinition-definitions.html#OperationDefinition.system){:target="_blank"}
```
Comment (MaxPhilips) [inline content/millennium/dstu2/operation-definition.md]: "operation _can_ be"

**Comment by MaxPhilips** [comment]
+1

**Comment by dennispatterson** [comment]
+1

**Comment by daliboz** [comment]
@dennispatterson @MaxPhilips take a quick look at the careplan corrections - @JFisk42 commented on the other PR just as I closed it out, and I noticed both URLs were a bit messed up.

**Comment by MaxPhilips** [comment]
+1

**Pull request merged**